src/bci_build/package/__init__.py

```diff
@@ -363,7 +363,8 @@ def oci_version(self) -> str:
     @property
     def build_name(self) -> str | None:
         if self.build_tags:
-            build_name: str = self.build_tags[0]
+            # build_tags[0] is with -RELEASE suffix, build_tags[1] without
+            build_name: str = self.build_tags[1]
```

Comment:
suggestion: assert it:
```suggestion
            build_name: str = self.build_tags[1]
            assert "RELEASE" not in build_name
```

Reply:
good idea, I added a variable for that and check for it now.